js/start.js

```diff
@@ -0,0 +1,13 @@
+console.log('javascript linked');
+
+$(document).ready(function() {
+  console.log('jQuery is ready!');
+
+  $('.homebutton').click(function(){
+    window.alert("you clicked the home button");
+
+    $('.like').click(function(){
```

Comment:
what your current code means:
1) When class ".homebutton" is clicked on...
    alert
    then (1) when class ".like" is clicked on, do something else.

It should be instead:
When class ".homebutton" is clicked on, do something
When class ".like" is clicked on, do something else